app/services/sign_in/assertion_validator.rb

```diff
@@ -32,7 +35,7 @@ def validate_iss
     end
 
     def validate_audience
-      unless decoded_assertion.aud.match(token_route)
+      unless Array(decoded_assertion.aud).include?(token_route)
```

Comment:
only saying this cause there are functions for the other decoded assertion attributes, so to be consistent, could you make a function that memoizes `aud`?